src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -350,6 +352,7 @@
   "InteractiveImport": "Interactive Import",
   "InteractiveSearch": "Interactive Search",
   "Interval": "Interval",
+  "IsAvailable": "Is Available",
```

Comment:
Should this be "Considered Available" to be more clear? Given its based on the set min availability.

Reply:
Yes,  I will change it.